components/draggable/README.md

```diff
@@ -0,0 +1,37 @@
+# Draggable
+
+`Draggable` is a ReactComponent that can wrap any React element to make it draggable. When used, a cross-browser (including IE) customisable drag image is created. The component clones the specified element on drag-start and uses the clone as a drag image during drag-over. Discards the clone on drag-end.
+
+## Props
+
+The component accepts the following props:
+
+### elementId
+
+The HTML id of the element to clone on drag
+
+- Type: `string`
+- Required: Yes
+
+### transferData
+
+The data object attached to the drag and drop event.
```

Comment:
What is the shape of the object? Is it arbitrary? If so, should we note that it's arbitrary?